packages/shared/styles/components/molecules/SfRadio.scss

```
@@ -56,6 +56,17 @@
     flex: 1;
     margin: var(--radio-content-margin, 0 0 0 var(--spacer-sm));
   }
+  &__details {
+    margin: var(--radio-details-margin, var(--spacer-2xs) 0 0 0);
```

```suggestion
    margin: var(--radio-details-margin, var(--spacer-xs) 0 0 0);
```
8px looks better than 4px

---

changed